packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -26,7 +24,7 @@ import '../runner/flutter_command.dart';
 /// Example:
 ///
 /// ```
-///   'linter': '0.1.35', // TODO(yjbanov): https://github.com/dart-lang/linter/issues/824
```

Comment:
The `TODO` in the example was intentional although perhaps a little "too real". We avoid manually pinning dependencies and when do we should either leave a TODO if the pinning it temporary or some other commend explaining why the pinning is necessary.

Reply:
Can we change it to something like `// See https://github...` instead?

Reply:
(Otherwise it artificially contributes to our tech debt calculations)

Reply:
We can probably just remove this example, since we now have a permanently-pinned package anyway.

Reply:
Done.

Reply:
Agreed that example is unnecessary in the presence of a permanently pinned package.